com.playeveryware.eos/Runtime/Core/PlatformManager.cs

```diff
@@ -99,21 +102,21 @@ public static Platform CurrentPlatform
 
         static PlatformManager()
         {
-            AddPlatformInfo(Platform.Android,     "Android",       "eos_android_config.json"    );
-            AddPlatformInfo(Platform.iOS,         "iOS",           "eos_ios_config.json"        );
-            AddPlatformInfo(Platform.Linux,       "Linux",         "eos_linux_config.json"      );
-            AddPlatformInfo(Platform.macOS,       "macOS",         "eos_macos_config.json"      );
-            AddPlatformInfo(Platform.Steam,       "Steam",         "eos_steam_config.json"      );
-            AddPlatformInfo(Platform.XboxOne,     "Xbox One",      "eos_xb1_config.json"        );
-            AddPlatformInfo(Platform.XboxSeriesX, "Xbox Series X", "eos_xsx_config.json"        );
-            AddPlatformInfo(Platform.PS4,         "PS4",           "eos_ps4_config.json"        );
-            AddPlatformInfo(Platform.PS5,         "PS5",           "eos_ps5_config.json"        );
-            AddPlatformInfo(Platform.Switch,      "Switch",        "eos_switch_config.json"     );
+            AddPlatformInfo(Platform.Android,     "Android",       "eos_android_config.json", typeof(AndroidConfig), "Android");
+            AddPlatformInfo(Platform.iOS,         "iOS",           "eos_ios_config.json",     typeof(IOSConfig),     "iPhone");
+            AddPlatformInfo(Platform.Linux,       "Linux",         "eos_linux_config.json",   typeof(LinuxConfig),   "Standalone");
+            AddPlatformInfo(Platform.macOS,       "macOS",         "eos_macos_config.json",   typeof(MacOSConfig),   "Standalone");
+            AddPlatformInfo(Platform.Steam,       "Steam",         "eos_steam_config.json",   null,         "Standalone");
+            AddPlatformInfo(Platform.XboxOne,     "Xbox One",      "eos_xb1_config.json",     null, "XboxOne");
+            AddPlatformInfo(Platform.XboxSeriesX, "Xbox Series X", "eos_xsx_config.json",     null, "XboxOne");
```

Comment:
Should the string here say "XboxOne" or something else? Perhaps there's a different phrase for it? Or is it the same one, because they all use similar GDK settings?

Reply:
Nope - these are the values that Unity uses to describe the icon within Unity that we can extract. Unity does not have a distinct logo for Xbox Series X vs Xbox One, so the same one is used.